src/System.ServiceModel.Primitives/ref/project.lock.json

```diff
@@ -2209,21 +2209,21 @@
         "lib/net46/System.Security.Cryptography.Primitives.dll",
         "lib/xamarinios10/_._",
         "lib/xamarinmac20/_._",
-        "ref/dotnet5.4/System.Security.Cryptography.Primitives.dll",
+        "ref/dotnet5.1/System.Security.Cryptography.Primitives.dll",
```

Comment:
I have no idea. In fact, I'm not sure how we're going to be able to update this file now as a normal build on windows will only update the other file. I'll try reverting all lock files and rebuilding again.